src/main/scala/loamstream/drm/DrmChunkRunner.scala

```diff
@@ -60,10 +59,9 @@ final case class DrmChunkRunner(
    */
   override def run(
       jobs: Set[LJob], 
-      jobOracle: JobOracle, 
-      shouldRestart: LJob => Boolean): Observable[Map[LJob, RunData]] = {
+      jobOracle: JobOracle): Observable[Map[LJob, RunData]] = {
```

Comment:
I think I asked/commented on this before in an earlier PR. But, is "Oracle" here meant to denote "all knowing" or is it literally referring to the company Oracle (i.e. it's a Java thing that needs to be used)?

Reply:
All Knowing, as before.